test/recurly.test.js

```diff
@@ -39,14 +40,26 @@ describe('recurly', () => {
       assert.equal(numDeps, 0, msg)
     })
   })
-  describe('#Client', () => {
+  describe('.Client', () => {
     it('Should expose the Client in the top level module', () => {
       assert.equal(recurly.Client, Client)
     })
   })
-  describe('#Account', () => {
+  describe('.Account', () => {
     it('Should expose resources in the top level module', () => {
       assert.equal(recurly.Account, Account)
     })
   })
+  describe('.errors', () => {
+    it('Should expose errors in the top level module', () => {
+      assert(Object.keys(recurly.errors).length > 0)
+      assert.equal(recurly.ApiError, ApiError)
+    })
+  })
+  describe('Schema.locateResource', () => {
+    it('Should implement Schema.locateResource', () => {
+      const clazz = recurly.Schema.locateResource('Account')
```

Comment:
heh, never seen it like this, always `klass`. I like it.

Reply:
I guess I picked this up from Java conventions.